tracetools/span.go

```diff
@@ -72,6 +83,39 @@ func (s *OpenTracingSpan) RecordError(err error) {
 	ext.LogError(s.Span, err)
 }
 
+type OpenTelemetrySpan struct {
+	Span trace.Span
+}
+
+func NewOpenTelemetrySpan(base trace.Span) *OpenTelemetrySpan {
+	return &OpenTelemetrySpan{Span: base}
+}
+
+// AddAttributes adds the given attributes to the OpenTelemetrySpan. Only string attributes are accepted.
+func (s *OpenTelemetrySpan) AddAttributes(attributes map[string]string) {
+	for k, v := range attributes {
+		s.Span.SetAttributes(attribute.String(k, v))
+	}
+}
+
+// FinishWithError adds error information to the OpentelemetrySpan if error isn't nil, and records the span as having finished
+func (s *OpenTelemetrySpan) FinishWithError(err error) {
+	s.RecordError(err)
+	s.Span.End()
+}
+
+// RecordError records an error on the given span.
+// Will panic if span is anything other than an opentracing.Span, a trace.Span, or nil
+// noop when span or error are nil
```

Comment:
Is this comment accurate? Doesn't `s` have to be an OpenTelemetrySpan?

Reply:
yeah, good point. this one is a leftover from renaming things, and they have such similar names that it's really easy to miss.